sephora/game/custom_turfs.dm

```diff
@@ -43,19 +94,63 @@
 	name = "durasteel hull plating"
 	icon = 'sephora/icons/turf/floors.dmi'
 	icon_state = "durasteel"
+	floor_tile = /obj/item/stack/tile/plasteel/ship
```

Comment:
What's this do?

Reply:
Lets you pry up the correct item with a crowbar?